test/content/audio/train_is_boarding_test.exs

```diff
@@ -48,7 +36,7 @@ defmodule Content.Audio.TrainIsBoardingTest do
 
       assert Content.Audio.to_params(audio) ==
                {:canned,
-                {"111", ["501", "21000", "507", "21000", "4016", "21000", "544", "21000", "542"],
+                {"111", ["501", "21000", "4016", "21000", "864", "21000", "544", "21000", "542"],
```

Comment:
Might be a nitpick, but could you do something here like in `realtime_test.exs` with the `expect_audios` function and check the `tts_value` of the `Audio`? It's difficult to parse what's changing in all of these tests when converting between the numbers and the corresponding text value.

Reply:
Not in this particular case, because this is testing `to_params`, which deals just with the numerical take ids. I agree it's a pain, but it's sort of an inherent pain with the current system. The way I've been verifying it is by searching the code for the take id, which pretty consistently gets you to a line that defines the mapping, so you know what it ends up saying. In this case, `"507" -> "train to", "4016" -> "Ashmont"` becomes `"4016" -> "Ashmont", "864" -> "train"`, which is the purpose of this change.